src/main/server.js

```diff
@@ -216,6 +216,56 @@ function createServer(options) {
     //     }
     server.post('/addMetaQuestion', service.addMetaQuestion);
 
+    //request: {
+    //       selectedMetaQuestion: MetaQuestion,
+    //       selectedKey: {answer: str, explanation: str },
+    //       selectedDistractors: [{answer: str, explanation: str }],
+    //     }
+    //response: {
+    //       id: num
+    //       stem: str,
+    //       key: {answer: str, explanation: str },
+    //       distractors: [{answer: str, explanation: str }],
+    //      appendix: {
+    //          title: str,
+    //          tag: str,
+    //          content: str
+    //       }
+    //     }
+    server.post('/addManualMetaQuestionToExam', service.addManualMetaQuestionToExam);
+
+
+    //request: {
+    //       selectedMetaQuestion: MetaQuestion,
+    //       selectedKey: null,
+    //       selectedDistractors: [] - empty array,
+    //     }
+    //response: {
+    //       id: num
+    //       stem: str,
+    //       key: {answer: str, explanation: str },
+    //       distractors: [{answer: str, explanation: str }],
+    //      appendix: {
+    //          title: str,
+    //          tag: str,
+    //          content: str
+    //       }
+    //     }
```

Comment:
having this documentation here isn't ideal as this area isn't really viewed/worked with.
moving it to the facade would be a better choice
and you can document only the returned value, and have the sent data be self documented with the call to `validateParameters`

Reply:
Agree, will be moved